### ISSUE TITLE
Selective regularization is causing sporadic build errors

### ISSUE THREAD

**Issue by kbattocchi** [microsoft/EconML#179]
The short term fix is to temporarily remove selective regularization from utilities and tests.

The longer term fix is to switch to the prototype implementation, which doesn't require tensorflow or work via gradient descent.
